lerna.json

```diff
@@ -1,6 +1,6 @@
 {
   "lerna": "2.9.1",
-  "version": "0.7.2",
+  "version": "0.8.0",
```

Comment:
This will be done automatically when you publish

Reply:
I guess our publish script would need to pass a parameter I am not quite familiar with how that works.